Add pgAdmin to PsotgreSQL

acwalter opened the pull request:
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/1736)

JPGWalterMS committed:
Add pgAdmin pt PsotgreSQL

acwalter commented:
@MarNicGit can you take a look? The error identifies a preexisting root,PostgreSQL but it is not available in master yet. This is adding a second folder to that root. My assumption is the root is getting flagged as a dupe when this should be a subfolder, not a new root. Here is the reference https://github.com/microsoft/winget-pkgs/pull/1728

MarNicGit commented on manifests/PostgreSQL/pgAdmin/4.22.0.0.yaml:
```diff
@@ -0,0 +1,20 @@
+Homepage: https://www.pgadmin.org/
+Tags: "pgsql, sql, postgre"
+InstallerType: exe
+Switches:
+    Silent: /verysilent
+    SilentWithProgress: /verysilent
+Installers:
+    Arch: x86
+    Url: https://ftp.postgresql.org/pub/pgadmin/pgadmin4/v4.22/windows/pgadmin4-4.22-x86.exe
+    Sha256: E219BB89752448A1599B7D52CD71E2EE6600D1E63508EFBC9409D109A5A86927
```
```suggestion
Installers:
  Arch: x86
  Url: https://ftp.postgresql.org/pub/pgadmin/pgadmin4/v4.22/windows/pgadmin4-4.22-x86.exe
  Sha256: E219BB89752448A1599B7D52CD71E2EE6600D1E63508EFBC9409D109A5A86927 
  Switches:
    Silent: /verysilent
    SilentWithProgress: /verysilent
```

Switches is not a root node and goes directly under the installer

acwalter committed:
Update manifests/PostgreSQL/pgAdmin/4.22.0.0.yaml

I am still unable to validate locally. 

Manifest Warning: Silent and SilentWithProgress switches are not specified for InstallerType exe. Please make sure the installer can run unattended.
Manifest Error: Duplicate installer entry found.

Co-authored-by: MarNicGit <47538428+MarNicGit@users.noreply.github.com>

MarNicGit commented on manifests/PostgreSQL/pgAdmin/4.22.0.0.yaml:
```diff
@@ -0,0 +1,20 @@
+Tags: "pgsql, sql, postgre"
+InstallerType: exe
+Installers:
+  Arch: x86
+  Url: https://ftp.postgresql.org/pub/pgadmin/pgadmin4/v4.22/windows/pgadmin4-4.22-x86.exe
+  Sha256: E219BB89752448A1599B7D52CD71E2EE6600D1E63508EFBC9409D109A5A86927 
+  Switches:
+    Silent: /verysilent
+    SilentWithProgress: /verysilent
```
Woops, forgot to add a dash
```suggestion
 - Arch: x86
   Url: https://ftp.postgresql.org/pub/pgadmin/pgadmin4/v4.22/windows/pgadmin4-4.22-x86.exe
   Sha256: E219BB89752448A1599B7D52CD71E2EE6600D1E63508EFBC9409D109A5A86927 
   Switches:
     Silent: /verysilent
     SilentWithProgress: /verysilent

```

acwalter committed:
Update manifests/PostgreSQL/pgAdmin/4.22.0.0.yaml

Ugh, I saw that in the spec and thought it was a typo. That all validates and installs locally now. Sorry to bug you I was going crazy.

Co-authored-by: MarNicGit <47538428+MarNicGit@users.noreply.github.com>
